css/properties/transition.json

```diff
@@ -0,0 +1,239 @@
+              {
+                "prefix": "-webkit-",
+                "version_added": "44",
+                "flag": {
+                  "type": "preference",
+                  "name": "layout.css.prefixes.webkit",
+                  "value_to_set": "true"
+                }
+              }
+            ],
+            "ie": {
+              "version_added": "10"
+            },
+            "ie_mobile": {
+              "version_added": "10"
+            },
+            "opera": [
+              {
+                "version_added": "12.1"
+              },
+              {
+                "prefix": "-o-",
+                "version_added": "10.1",
+                "version_removed": "12.5"
+              }
+            ],
+            "opera_android": [
+              {
+                "version_added": "12.1"
+              },
+              {
+                "prefix": "-o-",
+                "version_added": "10",
+                "version_removed": "12.5"
+              }
+            ],
+            "safari": [
+              {
+                "version_added": "6.1"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": "3"
+              }
+            ],
+            "safari_ios": {
+              "prefix": "-webkit-",
+              "version_added": "3.2"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "gradients": {
+          "__compat": {
+            "description": "Gradients",
+            "support": {
+              "webview_android": {
+                "version_added": false
+              },
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": {
+                "version_added": null
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": "10"
+              },
+              "ie_mobile": {
+                "version_added": "10"
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          },
+          "frames": {
```

Comment:
steps() [with 2 parameters] that is a timing-function that exists for a very long time. 
frames() was an experimental version, now abandoned, that is superseded by an extension of the syntax of steps(). This extension, once implemented, applies everywhere a timing-function is implemented.

I think we should only document bcd for it in types/timing-function.json (not yet created) and forget it here.

So my proposal is to remove the entry here.